deduplicator.py

```diff
@@ -83,6 +96,9 @@ def deduplicate():
     endpoints = config_json["endpoints"]
 
     exit_code = 0
+    summary = {}
     
     deduplicate()
+    pretty_print(summary)
```

Comment:
Could we add some whitespace before this summary text and also a heading such as: "SUMMARY". Before this method prints out text, there will be at least 4 different outputs of json.